tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -381,8 +384,25 @@ def test_huggingface_conversion_callback_interval(
 @pytest.mark.world_size(2)
 @pytest.mark.gpu
 @pytest.mark.parametrize(
-    'model,tie_word_embeddings',
-    [('mpt', True), ('mpt', False), ('neo', None), ('llama2', None)],
+    'model,tie_word_embeddings,peft_config',
```

Comment:
this test is getting kind of long, would be great to refactor. don't feel strongly if it's in this PR or as a follow up.

Reply:
deferring :)